README.md

```diff
@@ -165,4 +165,50 @@ Once you've done that, restart the IDE.
 * Run pmd for test
 ```shell
 ./gradlew pmdTest
+```
+
+### SONAR
+
+#### Install sonarqube docker image locally
+  - Make sure you have docker installed on your machine - 
```

Comment:
I think we can create another issues for the following: 

* create a docker compose 
* create Makefile 
* include steps how to configure and run via intelliJ, using sonar plugin

Reply:
Done